modules/attributes/pages/document-attributes-reference.adoc

```diff
@@ -26,7 +27,7 @@ This effective value is prefixed with an equals sign and enclosed in square brac
 The effective value will not be visible to an attribute reference.
```

Comment:
If it works exactly the same I think we should use the same wording, so either:

"The effective value can't be resolved using an attribute reference." or "An implied value will not be visible to an attribute reference".

Otherwise, it implies that there's a slight difference.

(sorry not really related to this change but I just noticed it)

Reply:
There is a slight difference. In the case of an effective value, the attribute reference will see the blank value. In the case of an implied value, the attribute reference won't resolve at all. But I can still find a way to explain this better.